components/scream/src/physics/shoc/tests/shoc_varorcovar_tests.cpp

```diff
@@ -270,6 +270,58 @@ struct UnitWrap::UnitTest<D>::TestShocVarorCovar {
     }
   }
 
+
+static void run_bfb()
+  {
+    SHOCVarorcovarData SDS_f90[] = {
+      //               shcol, nlev, nlevi
+      SHOCVarorcovarData(10, 71, 72, 1),
+      SHOCVarorcovarData(10, 12, 13, 1),
+      SHOCVarorcovarData(7,  16, 17, 1),
+      SHOCVarorcovarData(2, 7, 8, 0.005),
+    };
+
+    static constexpr Int num_runs = sizeof(SDS_f90) / sizeof(SHOCVarorcovarData);
+
+    for (Int i = 0; i < num_runs; ++i) {
+      SDS_f90[i].randomize();
+    }
+
+    // Create copies of data for use by cxx. Needs to happen before fortran calls so that
+    // inout data is in original state
+    SHOCVarorcovarData SDS_cxx[] = {
+      SHOCVarorcovarData(SDS_f90[0]),
+      SHOCVarorcovarData(SDS_f90[1]),
+      SHOCVarorcovarData(SDS_f90[2]),
+      SHOCVarorcovarData(SDS_f90[3]),
+    };
+
+    // Assume all data is in C layout
+
+    // Get data from fortran
+    for (Int i = 0; i < num_runs; ++i) {
```

Comment:
Use cxx11-style iteration.

Reply:
I consolidated all the outer-loops into 1. Let me know what you think.